plugins/scaffolder-backend-module-backstage-rhaap/src/actions/schemas/rhaapActionPayloadUtils.test.ts

```
@@ -0,0 +1,691 @@
+});
+
+describe('launchJobTemplateInputSchema', () => {
+  it('preprocesses and parses a minimal valid launch payload', () => {
+    const result = launchJobTemplateInputSchema.safeParse({
+      template: 'My template',
+      job_type: 'check',
+      inventory: { id: 1, name: 'Inv' },
+    });
+    expect(result.success).toBe(true);
+    if (result.success) {
+      expect(result.data.template).toBe('My template');
```

ESLint

[error] Avoid calling `expect` conditionally`

(jest/no-conditional-expect)